helm-go-lib/main.go

```diff
@@ -24,14 +24,14 @@ import (
 	Example command line to run from within the directory this file is in:
 		go run main.go \
 			-logtostderr=true -stderrthreshold=INFO \
-			my-namespace my-kube-token-string https://12.34.567.890 ./ca_file bitnami/nginx key1=v1,key2.key3=v2
+			my-namespace my-kube-token-string https://12.34.567.890 ./ca_file bitnami/nginx 6.0.5 key1=v1,key2.key3=v2
 */
 func main() {
 	flag.Parse()
 	progArgs := flag.Args()
 	lenProgArgs := len(progArgs)
 
-	if (lenProgArgs != 6) && (lenProgArgs != 7) {
+	if (lenProgArgs != 7) && (lenProgArgs != 8) {
 		fmt.Println("Expected args: <namespace> <kube token> <api server> <ca file> <release name> <chart name> <values>",
```

Comment:
should this print be updated too? I don't see anything indicated that theres an optional param

Reply:
Yes! Thanks for the catch.